src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ShowColumns.txt

```diff
@@ -1,4 +1,4 @@
-﻿#SETUP: SupportColumnNamesAsIdentifiers
+﻿#SETUP: SupportColumnNamesAsIdentifiers, MutationFunctionsTestSetup
 
 >> ShowColumns(Table({a:1,b:2},{a:3,b:4}), b)
```

Comment:
Instead of adding this new setup handler here, can you please create another test file? Power Apps doesn't support the mutation functions test set up, so that will make it not able to run any of the tests